vector/v.colors/tests/test_v_colors.py

```diff
@@ -0,0 +1,284 @@
+import os
+import re
+import pytest
+import grass.script as gs
+import grass.exceptions
+
+
+@pytest.fixture
+def simple_vector_map(tmp_path):
+    """Fixture to create a simple vector map with 3 points and a numeric attribute column 'val'."""
+    project = tmp_path / "v_colors_project"
+    gs.create_project(project)
+
+    with gs.setup.init(project, env=os.environ.copy()) as session:
+        # Set the computational region
+        gs.run_command("g.region", n=10, s=0, e=10, w=0, res=1, env=session.env)
+
+        # Create a small ASCII file with coordinates
+        coords = "1|1\n2|2\n3|3"
+        path = tmp_path / "coords.csv"
+        path.write_text(coords)
+
+        # Import the points into a vector map
+        gs.run_command(
+            "v.in.ascii",
+            input=str(path),
+            output="test_points",
+            separator="|",
+            x=1,
+            y=2,
+            overwrite=True,
+            env=session.env,
+        )
+
+        # Add a new attribute table and column to hold numeric values
+        gs.run_command("v.db.addtable", map="test_points", env=session.env)
+        gs.run_command(
+            "v.db.addcolumn",
+            map="test_points",
+            columns="val DOUBLE PRECISION",
+            env=session.env,
+        )
+
+        # Update the 'val' column for each feature
+        for i, val in enumerate([10, 20, 30], start=1):
+            gs.run_command(
+                "v.db.update",
+                map="test_points",
+                column="val",
+                value=str(val),
+                where=f"cat = {i}",
+                env=session.env,
+            )
+
+        yield "test_points", session
```

Comment:
@petrasovaa Is there something that could help with the speed of the setup phase?
It ends up having 11x 3.2-3.5 seconds, so still 35 seconds only of setup time just for this test file due to that fixture. Probably one backend tool is longer than others to run.

Reply:
I'm sad to hear that it is so bad. Is the timing from Windows?

Since there are points, v.in.ascii can read them with attributes. Please, see the documentation:

https://grass.osgeo.org/grass-stable/manuals/v.in.ascii.html
https://grass.osgeo.org/grass-devel/manuals/v.in.ascii.html

This will cut down the number of subprocesses created.